app/shared/event/event_bus.py

```diff
@@ -0,0 +1,29 @@
+class EventBus():
+    def __init__(self):
+        self._subscribers = {}
+
+    async def publish(self, event):
+        event_classes = [x for x in list(
+            map(lambda x: x.__name__, type(event).mro())) if x != 'object']
```

Comment:
x is not a good variable name at all.
remove ambiguity? :
```suggestion
        event_classes = [x for x in list(
            map((lambda x: x.__name__), type(event).mro())) if x != 'object']
```
why map? :
```suggestion
        event_classes = [x for x in
                         [x.__name__ for x in type(event).mro()] 
                         if x != 'object']
```

Simpler solution:
```suggestion
        event_classes = [x.__name__ for x in type(event).mro() 
                         if x.__name__ != 'object']
```